docs/index.md

```diff
@@ -0,0 +1,15 @@
+# Introduction to Docker Volume Driver for vSphere
+
+Docker volume driver is designed to solve persistency needs for 
+stateful containers running on top of VMware's stack.
+
+Some of the high level features for the plugin are: 
+
+1. Designed to run over shared storage in a cluster (Single node 
+test setups supported as well)
+2. Easy to deploy and manage. There is zero configuration and 
+zero credential management.
+3. Integrated with vCenter and Photon Controller
+4. 
```

Comment:
Yeah the entire documentation needs updating, captured here https://github.com/vmware/docker-volume-vsphere/issues/552
Will remove the empty step for now.